src/plugins/maps_ems/common/ems_defaults.ts

```diff
@@ -8,7 +8,7 @@
 
 export const DEFAULT_EMS_FILE_API_URL = 'https://vector.maps.elastic.co';
 export const DEFAULT_EMS_TILE_API_URL = 'https://tiles.maps.elastic.co';
-export const DEFAULT_EMS_LANDING_PAGE_URL = 'https://maps.elastic.co/v8.7';
+export const DEFAULT_EMS_LANDING_PAGE_URL = 'https://maps.elastic.co/';
```

Comment:
Will there ever be (or need to be) links to branch-date versions like `8.10-2023-10-31` appended to this constant? Is it expected that those paths are generated dynamically?